ide/extbrowser/src/org/netbeans/modules/extbrowser/NbDdeBrowserImpl.java

```diff
@@ -274,7 +272,7 @@ private String realDDEServer () {
             /** if not found with getDefaultWindowsOpenCommand function
              *  fallback to previous method
              */
-            if (cmd.isEmpty())
+            if (cmd.isEmpty() || cmd == null)
```

Comment:
`cmd.isEmpty()` will throw a NPE when cmd is null

Reply:
bugger, does the git global config at least resolve the line ending issue?